### ISSUE TITLE
CSV Import: cannot open files with commas as decimal point

### ISSUE THREAD

**Issue by Scindix** [narunlifescience/AlphaPlot#28]
Edit: I'm using Version 1.011A-alpha

Thanks for making a modern version of qtiPlot.
There's just one issue I'd like to point out: I'm unable to import files that use commas as decimal seperator which is a format used in some European languages.
This is the csv file:
```
$ cat test.csv
Zeit t / s;Spannung U_B1 / V;Spannung U_A1 / V
0;0,665;-0,375
0,1;0,77;-0,29
0,2;0,855;-0,225
0,3;0,905;-0,2
0,401;0,91;-0,205
0,501;0,89;-0,245
0,6;0,83;-0,32
0,7;0,74;-0,41
0,801;0,63;-0,515
0,9;0,505;-0,625
1;0,385;-0,73
```
This is how I'm importing it:
![Screenshot from 2021-04-30 22-33-21](https://user-images.githubusercontent.com/8752119/116751534-78e81980-aa04-11eb-80b2-7cc85692e13e.png)
↑ Please note that I'm using the correct Decimal Seperators.
However the imported table consists mostly of zeros.
![Screenshot from 2021-04-30 22-37-32](https://user-images.githubusercontent.com/8752119/116751677-ad5bd580-aa04-11eb-8c40-d5c1283f2fc0.png)
If I perform the exact same import in SciDAVis it is imported correctly:
![Screenshot from 2021-04-30 22-41-09](https://user-images.githubusercontent.com/8752119/116752058-3b37c080-aa05-11eb-8c7a-d28ce31362b6.png)

**Comment by filipestevao** [comment]
Hi @Scindix,

Thanks for your collaboration. I can confirm this bug.

There is a way to get around this situation without having to modify the raw data.

First, the `Decimal Separators` is set to `1 000,0` or `1.000,0` (Edit → Preferences → General → Numeric Format).

After, I don't check the `Numeric data` in import window:

![alphaplot1](https://user-images.githubusercontent.com/50991168/117096023-4f99f700-ad3e-11eb-980e-f361f87a4731.png)

The data is imported as **Text**:

![alphaplot2](https://user-images.githubusercontent.com/50991168/117096039-545eab00-ad3e-11eb-9d41-2bd1848214e0.png)

Then, I select all the columns and in `Type` I set `Numeric` and `Apply`.

![alphaplot3](https://user-images.githubusercontent.com/50991168/117096051-5a548c00-ad3e-11eb-8a35-0282e41a66d3.png)

Of course, this does not remove the fact that this issue needs to be fixed. But it is an optional solution until it is done.

**Comment by narunlifescience** [comment]
Thank you for reporting. I will have a look at it later.

**Comment by narunlifescience** [comment]
fixed in new release v1.02
